frontend/src/metabase/visualizations/visualizations/CartesianChart/CartesianChart.tsx

```diff
@@ -75,6 +81,56 @@ export function CartesianChart({
 
   const eventHandlers: EChartsEventHandler[] = useMemo(
     () => [
+      {
+        eventName: "mouseout",
+        query: "series",
+        handler: () => {
+          onHoverChange?.(null);
+        },
+      },
+      {
+        eventName: "mousemove",
+        query: "series",
+        handler: event => {
```

Comment:
Can we add a type for the `event` parameter? I remember during prototyping it was difficult to write event handler logic without a type, for example below `event.event.event.target.nodeName` is intuitive to figure out without a type

Reply:
I added types here https://github.com/metabase/metabase/pull/37064/files#diff-c2a99fb424ee53cff58e8bfc769af58dd4e5f4636a5a2a303ba5912a953536ac